### PR TITLE
Converted to Keras Core: DeepLabV3Plus

### PR DESCRIPTION
**Pull request by anas-rz**
Update imports
change tf.keras to keras
repace tensorflow operations with keras ops
[Colab Notebook](https://colab.research.google.com/drive/1uk-k_AE1WDzxoiZOQARVzUv0IHIDnfVA?usp=sharing)
[deeplabv3_plus_keras_core.diff.txt](https://github.com/keras-team/keras-core/files/12098216/deeplabv3_plus_keras_core.diff.txt)

### CODE REVIEW

File: examples/keras_io/tensorflow/vision/deeplabv3_plus.py [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+import keras_core as keras
+from keras_core import layers
+from keras_core import ops
+
+import cv2
+import numpy as np
+from glob import glob
+from scipy.io import loadmat
+import matplotlib.pyplot as plt
+import tensorflow as tf
```
Comment (fchollet) [inline examples/keras_io/tensorflow/vision/deeplabv3_plus.py:45]: You can instead just import `from tensorflow import image as tf_image` and `from tensorflow import data as tf_data`, then move the example to the backend agnostic folder. TF is only needed for the data preprocessing, external to Keras.

File: examples/keras_io/tensorflow/vision/deeplabv3_plus.py [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+"""
+## Downloading the data
+
+We will use the [Crowd Instance-level Human Parsing Dataset](https://arxiv.org/abs/1811.12596)
+for training our model. The Crowd Instance-level Human Parsing (CIHP) dataset has 38,280 diverse human images.
+Each image in CIHP is labeled with pixel-wise annotations for 20 categories, as well as instance-level identification.
+This dataset can be used for the "human part segmentation" task.
+"""
+import os
+os.environ['KERAS_BACKEND'] = 'tensorflow'
```
Comment (fchollet) [inline examples/keras_io/tensorflow/vision/deeplabv3_plus.py:34]: Run `black` on the file.

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

**Comment by anas-rz** [comment]
Please check.

**Comment by anas-rz** [comment]
Moved preprocessing to model and replaced tensorflow with `tf_image`, `tf_data` and `tf_io` and run black on code.

**Review by fchollet — APPROVED** [review]
LGTM, thank you!

**Pull request merged**